src/schema.rs

```diff
@@ -1255,4 +1396,156 @@ $ rust-cmd1
         replace_lines(&mut actual, line_nums, replacement).unwrap();
         assert_eq!(expected, actual);
     }
+
+    #[test]
+    fn overwrite_toml_status_success() {
+        let expected = r#"
+bin.name = "cmd"
+"#;
+        let actual = overwrite_toml_status(
+            exit_code_to_status(0),
+            r#"
+bin.name = "cmd"
+status = "failed"
+"#
+            .into(),
+        )
+        .unwrap();
+        assert_eq!(expected, actual);
+    }
+
+    #[test]
+    fn overwrite_toml_status_failed() {
+        let expected = r#"
+bin.name = "cmd"
+status = { code = 1 }
+"#;
+        let actual = overwrite_toml_status(
+            exit_code_to_status(1),
+            r#"
+bin.name = "cmd"
+"#
+            .into(),
+        )
+        .unwrap();
+        assert_eq!(expected, actual);
+    }
+
+    #[test]
+    fn overwrite_toml_status_keeps_style() {
+        let expected = r#"
+bin.name = "cmd"
+status.code= 1
```

Comment:
The toml formatting is off, it should be ` = ` (leading space)

Reply:
I did that intentionally to make sure this preserves style

Reply:
Oh yes, the `actual` also has it.  Stood out to me because that is a specific case that is easy to mess up with `toml_edit` :).  A more clear way might be to have a comment.

Reply:
(granted, comments are likely to be invalidated by a change in the value)